.gitignore

```
@@ -102,3 +102,9 @@ dist
 
 # TernJS port file
 .tern-port
+
+# InteliJ's
+.idea/*
```

Also exclude root intellij `.iml` file from root

```suggestion
.idea/*
/*.iml
```